src/components/chart/_macro.spec.js

```diff
@@ -1648,6 +1665,19 @@ describe('Macro: Chart', () => {
                     expect($('[data-invalid-chart-type]').length).toBe(0);
```

Comment:
I think this needs to be updated as I have said in a previous comment about using classes

Reply:
Can we move this to the code tidy up ticket? There's a lot of feedback coming in here that isn't relevant to the PR and this is blocking other work.

Reply:
Yeah thats fair